### PR TITLE
Issue#798

### PR DESCRIPTION
**Pull request by reactexcel**

### CODE REVIEW

File: frontend/src/components/leave/manageLeaves/LeaveColorReference.jsx [hunk 1]
```diff
@@ -5,76 +5,78 @@ import { Row, Col, Nav, NavItem, NavLink } from "reactstrap";
 import classnames from "classnames";
 
 const LeaveColorReference = ({ filterLeaveList, selectedTab, userRole }) => {
-  if (userRole === CONFIG.ADMIN) {
+  // if (userRole === CONFIG.ADMIN) {
```
Comment (xmagestore) [inline frontend/src/components/leave/manageLeaves/LeaveColorReference.jsx]: why is this commented?

File: frontend/src/components/leave/manageLeaves/LeaveColorReference.jsx [hunk 2]
```diff
@@ -5,76 +5,78 @@ import { Row, Col, Nav, NavItem, NavLink } from "reactstrap";
...
     return (
-          <Nav className="bg-light manage-users-nav-bar" tabs>
-            <NavItem>
-              <NavLink
-                className={classnames({
+          <Nav className="status_header bg-body manage-users-nav-bar d-flex" tabs>
+            <NavItem className = 'flex-fill text-center'>
+              <NavLink id = 'header_point_1'
```
Comment (xmagestore) [inline frontend/src/components/leave/manageLeaves/LeaveColorReference.jsx]: why point1 point2 point3

File: frontend/src/components/leave/manageLeaves/LeaveColorReference.jsx [hunk 3]
```diff
@@ -5,76 +5,78 @@ import { Row, Col, Nav, NavItem, NavLink } from "reactstrap";
...
+  // }
+  // return (
+  //       <Nav className="bg-light manage-users-nav-bar" tabs>
+  //         <NavItem>
+  //           <NavLink
+  //             className={classnames({
+  //               active: selectedTab === "Pending"
+  //             })}
+  //             onClick={() => {
+  //               filterLeaveList("Pending");
```
Comment (xmagestore) [inline frontend/src/components/leave/manageLeaves/LeaveColorReference.jsx]: what is all this?? why is there so much commented code. either remove it or keep it

File: frontend/src/themeFlatkit/scripts/data.json [hunk 4]
```diff
@@ -24,7 +28,8 @@
         "color": {
 	        "primary":      "#0cc2aa",
 	        "accent":       "#a88add",
-	        "warn":         "#fcc100"
+          "warn":         "#fcc100",
+          "backyellow":   "#fff582"
```
Comment (xmagestore) [inline frontend/src/themeFlatkit/scripts/data.json]: why is this changed?

File: frontend/src/styles/leave/manageLeaves.scss [hunk 5]
```diff
@@ -57,6 +57,134 @@
...
 }
+.back-widget
+.status_header > .nav-item .nav-link.active {
+  box-shadow: none !important;
+}
+#header_point_1{
+  border-bottom: 3px solid;
+}
+#header_point_2 {
+  border-bottom: 3px solid;
```
Comment (xmagestore) [inline frontend/src/styles/leave/manageLeaves.scss]: all 3 classes look the same

File: frontend/src/modules/leave/components/manageLeaves/ViewLeave.jsx [hunk 6]
```diff
@@ -146,35 +213,101 @@ class ViewLeave extends React.Component {
...
                   </div>
                 }
               </div>
-              <div>Applied On <i><b>{this.props.selectedLeave.applied_on}</b></i></div>
-              <div><b>{this.props.selectedLeave.from_date} To {this.props.selectedLeave.to_date}</b></div>
-              <div>No. of Days - <i><b>{this.props.selectedLeave.no_of_days}</b></i></div>
-              {this.props.selectedLeave.day_status === '' ? '' : <div>Leave apply for - <i><b>{this.props.selectedLeave.day_status === '1' ? 'First Half' : 'Second Half'}</b></i></div>}
-              <div className="word-break-all">Reason - <i><b>{this.props.selectedLeave.reason}</b></i></div>
+              {/* <div>Applied On <i><b>{this.props.selectedLeave.applied_on}</b></i></div> */}
+              {/* <div><b>{this.props.selectedLeave.from_date} To {this.props.selectedLeave.to_date}</b></div> */}
```
Comment (xmagestore) [inline frontend/src/modules/leave/components/manageLeaves/ViewLeave.jsx]: too much commented code in entire file?

File: frontend/src/themeFlatkit/scripts/data.json (hunk 4 above)
Comment (reactexcel) [reply frontend/src/themeFlatkit/scripts/data.json]: i just put a comma , rest will same.

File: frontend/src/modules/leave/components/manageLeaves/ViewLeave.jsx [hunk 7]
```diff
@@ -26,11 +27,18 @@ class ViewLeave extends React.Component {
...
   }
   handleSave = data => {
-    this.props.onAddDescription(this.props.selectedLeave.id, this.state.messageByHr, data);
-    this.setState({
-      edit:        false,
-      messageByHr: ''
-    });
+    if(this.state.messageByHr) {
+      this.props.onAddDescription(this.props.selectedLeave.id, this.state.messageByHr, data);
+    if(data == 1 || data == 2){
```
Comment (xmagestore) [inline frontend/src/modules/leave/components/manageLeaves/ViewLeave.jsx]: what is this data ==1 or 2 . write in code comments

File: frontend/src/components/leave/manageLeaves/LeaveColorReference.jsx (hunk 1 above)
Comment (reactexcel) [reply frontend/src/components/leave/manageLeaves/LeaveColorReference.jsx]: i resolved it as per design admin-role and admin role. the design is same so i put condition if role is either admin or hr this whole thing see.
